### ISSUE TITLE
Space for OS X buttons needs to go for WIndows/Linux

### ISSUE THREAD

**Issue by Tyriar** [zeit/hyper#113]
Ubuntu 16.04

![image](https://cloud.githubusercontent.com/assets/2193314/16889420/2c1ec162-4a9b-11e6-9973-8d0be31e2bce.png)

Clicking it also causes an error:

```
header.js?f251:62 Uncaught TypeError: Cannot read property 'emit' of undefinedonHeaderMouseDown @ header.js?f251:62ReactErrorUtils.invokeGuardedCallback @ ReactErrorUtils.js?6b31:70executeDispatch @ EventPluginUtils.js?0958:89executeDispatchesInOrder @ EventPluginUtils.js?0958:112executeDispatchesAndRelease @ EventPluginHub.js?c6b1:44executeDispatchesAndReleaseTopLevel @ EventPluginHub.js?c6b1:55forEachAccumulated @ forEachAccumulated.js?b898:25processEventQueue @ EventPluginHub.js?c6b1:221runEventQueueInBatch @ ReactEventEmitterMixin.js?8a20:18handleTopLevel @ ReactEventEmitterMixin.js?8a20:29handleTopLevelImpl @ ReactEventListener.js?2365:73perform @ Transaction.js?6dff:138batchedUpdates @ ReactDefaultBatchingStrategy.js?ef70:63batchedUpdates @ ReactUpdates.js?ce09:98dispatchEvent @ ReactEventListener.js?2365:150
```

**Comment by flybayer** [comment]
This is fixed for me on master with Ubuntu 14.04

![image](https://cloud.githubusercontent.com/assets/8813276/19165405/4264b606-8bc9-11e6-9907-a5889957c574.png)

**Comment by matheuss** [comment]
Was fixed by #524 😄
